packages/lib-utils/src/components/table/VirtualizedTableBody.tsx

```diff
@@ -38,6 +36,18 @@ export type TableRowProps = {
   className?: string;
 };
 
+export const Td = React.forwardRef<
```

Comment:
@fhlavac is there any added functionality to exposing this component or can apps/plugins just use the `PFTd` component exposed directly from patternfly?

Reply:
@florkbr our Td component has additional classes which are trying to fix the styling issues. There is no other reason but also probably no better solution for now.